src/main/java/com/armory/logsort/LogSortApplication.java

```diff
@@ -13,14 +14,46 @@ public class LogSortApplication {
 
     public static void main(String[] args) {
         // generate log files
-        new LogGenerator(100, 100).generate("./src/main/resources/generated/logs/");
+        // commented to test without generating the files
+        new LogGenerator(1000, 1000).generate("./src/main/resources/generated/logs/");
```

Comment:
change this to `new LogGenerator(100, 100).generate("./src/main/resources/generated/logs/");` to test the solution with a smaller sample size, the `1000` values will generate around 48MBs of logs as a stress-test.